Specta/Specta/SpectaDSL.m

```diff
@@ -61,10 +61,6 @@ void spt_itShouldBehaveLike_(NSString *fileName, NSUInteger lineNumber, NSString
         afterEach(^{
           [dataDict removeAllObjects];
         });
-
-        afterAll(^{
```

Comment:
WAT

Reply:
As discussed, this is required for Specta's test suite to pass.